config/v3_1_experimental/types/schema.go

```diff
@@ -202,7 +190,7 @@ type Systemd struct {
 }
 
 type TLS struct {
-	CertificateAuthorities []CaReference `json:"certificateAuthorities,omitempty"`
+	CertificateAuthorities []Resource `json:"certificateAuthorities,omitempty"`
```

Comment:
Just to make sure, let me ask this directly: it is fine to completely change this public type (and the rest in this package too) without an actual semver break only because it belongs to an "_experimental" subpackage, right?

Reply:
That's my interpretation, yeah.